src/renderer/SpeechBubbleSkin.ts

```diff
@@ -24,27 +39,32 @@ export default class SpeechBubbleSkin extends Skin {
     this.height = 0;
     this.offsetX = -bubbleStyle.strokeWidth / 2;
     this.offsetY = this.offsetX + bubbleStyle.tailHeight;
-
-    this._renderBubble(this._bubble);
   }
 
   // To ensure proper text measurement and drawing, it's necessary to restyle the canvas after resizing it.
-  _restyleCanvas() {
-    const ctx = this._canvas.getContext("2d");
+  private _restyleCanvas(): void {
+    const ctx = this._ctx;
     ctx.font = "16px sans-serif";
     ctx.textBaseline = "hanging";
   }
 
-  set flipped(flipped) {
+  public set flipped(flipped: boolean) {
```

Comment:
Recurring theme a bit in this PR, but as usual I don't understand why some property setters get type annotations and others don't - especially these really simple ones where I assumed you're sometimes (but not always?) inferring the type for the value from the type of the property being set to it.

Reply:
The type annotation can be inferred only if there's a corresponding getter. In this case there wasn't; I'll add one.